README.md

```diff
@@ -136,6 +136,51 @@ Follow the official VS Code MCP server setup guide and add the Next.js DevTools
 
 </details>
 
+<details>
+<summary>GitHub Copilot Coding Agent</summary>
+
+Add the following configuration to the `mcp` section of your GitHub Copilot Coding Agent configuration file Repository->Settings->Copilot->Coding agent->MCP configuration:
+
+```json
+{
+  "mcpServers": {
+    "next-devtools": {
+      "type": "local",
+      "command": "npx",
+      "args": ["-y", "next-devtools-mcp@latest"],
+      "tools": ["init", "nextjs_docs", "browser_eval", "nextjs_runtime", "upgrade_nextjs_16", "enable_cache_components"]
```

Comment:
can we not specify the tools as there might be changes for the tools, we'd like they to be discovered by agent
```suggestion
```